locales/en.json

```diff
@@ -80,17 +81,73 @@
 		}
 	},
 	"chargeView": {
-		"expiry": "Expiry date",
-		"cvc": "Card security code",
+    "title": "Enter card details",
+    "expiry": "Expiry date",
+    "expiryHint": "For example, 10/20",
+    "expiryMonth": "Month",
+    "expiryYear": "Year",
+    "cvc": "Card security code",
+    "cvcTip": "The last 3 digits on the back of the card",
+    "amexcvcTip": "The last 4 digits after the card number on the front",
+    "amexcvcNonjs": "For American Express,",
+    "or": "or",
 		"cardNo": "Card number",
-		"cardholderName": "Name on card",
+    "cardholderName": "Name on card",
+    "billingAddress": "Billing address",
+    "billingAddressHint": "This is the address associated with the card",
 		"building": "Building number or name and street",
 		"city": "Town or city",
 		"postcode": "Postcode",
 		"country": "Country or territory",
-		"email": "Email",
-		"cvcTip": "The last 3 digits on the back of the card",
-		"amexcvcTip": "The last 4 digits after the card number on the front",
-		"amexcvcNonjs": "For American Express,"
-	}
+    "email": "Email",
+    "emailHint": "We’ll send your payment confirmation here",
+    "emailConfirmation": "An email will be sent to:",
+    "paymentSummary": "Payment summary",
+    "totalAmount": "Total amount:",
+    "continueButton": "Continue",
+    "cancelButton": "Cancel payment"
+  },
+  "confirmView": {
+    "title": "Confirm your payment",
+    "card": "Card number:",
+    "expiry": "Expiry date:",
+    "name": "Name on card:",
+    "address": "Billing address:",
+    "email": "Confirmation email:",
+    "confirmButton": "Confirm payment",
+    "cancelButton": "Cancel payment"
+  },
+  "errorViews": {
+    "either": "You can either:",
+    "continue": "Continue with your payment",
+    "tryAgain": "Go back to try the payment again",
+    "problem": "There was a problem with your payment",
+    "declined": "Your payment has been declined",
+    "contactYourBank": "Contact your bank for more details. No money has been taken from your account.",
+    "paymentStatus": "Your payment was",
+    "unsuccessful": "Your payment was unsuccessful",
+    "noMoneyTaken": "No money has been taken from your account.",
+    "toMakeChanges": "To make any changes to your payment you’ll need to contact the service.",
+    "viewSummary": "View your payment summary",
+    "paymentExpired": "Your Payment expired",
+    "sessionExpired": "Your payment session has expired",
+    "paymentCancelled": "Your Payment was cancelled",
+    "paymentCancelledTitle": "Payment cancelled",
+    "paymentCancelledDescription": "Your payment has been cancelled by the government service. This may have happened for a number of reasons.",
+    "error": "An error occurred",
+    "technicalProblems": "Sorry, we’re experiencing technical problems"
+  },
+  "otherViews": {
```

Comment:
With the exception of `inProgress`, `userCancelledPayment` and `goBackToService`, I think these are all basically shown during the authorisation part of the process (including 3D Secure), so maybe we could move most of them into a `i18n.authorisation` object and try to find homes for the others (I’m worried that `otherViews` will just become a grab bag where people put anything and everything).

(Yes, I realise that some of the strings are shown in the `capture_waiting.njk` page but I think that page dates back to before we had asynchronous capture and could probably be removed at some point.)